### PR TITLE
tidb-binlog: update kafka-max-message-size configuration

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
tidb-binlog: update kafka-max-message-size configuration

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7445
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-binlog/deploy-tidb-binlog.md [hunk 1]
```diff
@@ -370,9 +370,12 @@ The following part shows how to use Pump and Drainer based on the nodes above.
         # zookeeper-addrs = "127.0.0.1:2181"
         # kafka-addrs = "127.0.0.1:9092"
         # kafka-version = "0.8.2.0"
+        # The maximum number of messages (number of binlogs) in a broker request. If it is left blank or a value smaller than 0 is configured, the default value 1024 is used.
```
Comment (lichunzhu) [inline tidb-binlog/deploy-tidb-binlog.md:373]: not sure
```suggestion
        # The maximum number of messages (number of binlogs) in a broker request. If it is left blank or a value less than 0 is configured, the default value 1024 will be used.
```

**Review by lichunzhu — APPROVED** [review]
LGTM

File: tidb-binlog/deploy-tidb-binlog.md (hunk 1 above)
Comment (shichun-0415) [reply tidb-binlog/deploy-tidb-binlog.md:373]: > not sure

Thanks for your suggestion. 
1. Both 'smaller than' and 'less than' are correct here.
2. In technical documents, we seldom use the future tense. In 'if' clauses, if the meaning is clear, we also prefer the present tense.

File: tidb-binlog/deploy-tidb-binlog.md [hunk 2]
```diff
@@ -370,9 +370,12 @@ The following part shows how to use Pump and Drainer based on the nodes above.
...
         # kafka-max-messages = 1024
+        # The maximum size of a broker request (unit: byte). The default value is 1 GiB and the maximum value is 2 GiB.
+        # kafka-max-message-size = 1073741824
```
Comment (hfxsd) [inline tidb-binlog/deploy-tidb-binlog.md:376]: 可以加句话解释一下 1073741824 byte 是多少 GB

**Review by hfxsd — APPROVED** [review]

File: tidb-binlog/deploy-tidb-binlog.md (hunk 2 above)
Comment (shichun-0415) [reply tidb-binlog/deploy-tidb-binlog.md:376]: 刚跟 chunzhu 交流了，用户应该知道 1073741824 byte 是1 GB，这里就不额外解释了。

**Pull request merged**